pkg/api/types.go

```diff
@@ -167,6 +167,19 @@ type Config struct {
 	// the container that runs assemble.
 	// All files we inject will be truncated after the assemble script finishes.
 	Injections InjectionList
+
+	// CGroupLimits describes the cgroups limits that will be applied to any containers
```

Comment:
cgroups is the name of the technology.  so it's cgroups limits that are being enforced.

the struct is the limits for a particular cgroup, so it's cgrouplimits (the limits for this cgroup)